attestation-evidence-broker/docs/design/design.md

```diff
@@ -0,0 +1,121 @@
+This document provides that how to use Attestation Evidence Broker (AEB) which is a host service to satisfy the requests from guest for getting **SEV(-ES) runtime remote attestation report**.
+
+# Background of SEV(-ES) runtime remote attestation
+
+Compared with SEV SNP and Intel TDX, the runtime remote attestation of SEV(-ES) does NOT support guest-provided data to be included into the attestation report [1,2]. In addition, SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values. Although SEV(-ES) has this limitation, runtime remote attestation is still useful. When the SEV(-ES) guest is running, it can report the attestation evidence with fixed launch digest as a heartbeat for trusted healthy check.
+
+SEV(-ES) remote attestation includes two parties: attester and verifier.
+
+- The attester running in the SEV(-ES) guest is responsible for collecting attestation evidence.
+- The verifier runs in the trusted domain of the tenant and is responsible for verifying the attestation evidence sent by the attester. Noted that **verifier can run on any platform (including non-SEV(-ES) platforms)**.
+
+What SEV(-ES) attestation evidence should be collected by the attester?
+
+# The content of SEV(-ES) Attestation Evidence
+
+Verifier uses the following SEV(-ES) certificate chain to verify the integrity of the attestation report generated by the `ATTESTATION` command [2]:
+```bash
+ARK -> ASK -> CEK -> PEK -> report
+OCA -> PEK -> report
+```
+
+- `foo -> bar` means to use foo's public key to verify the integrity of bar.
```

Comment:
s/use foo's public key to verify the integrity of bar./use the public key of `foo` to verify the signature of `bar`./